betamax/serializers/proxy.py

```diff
@@ -27,6 +27,13 @@ def __init__(self, serializer, cassette_path, allow_serialization=False):
         self.cassette_path = cassette_path
 
     def _ensure_path_exists(self):
+        directory, _ = os.path.split(self.cassette_path)
+        if directory != '':
+            if not os.path.isdir(directory):
+                raise OSError(
```

Comment:
Yeah, I feel a little odd raising an OSError. Maybe add a custom Betamax error just to avoid confusion as to where this error is coming from? I personally wouldn't expect a library to raise an OSError directly (or really a RuntimeError)